home.py

```diff
@@ -0,0 +1,88 @@
+from crypt import methods
+from flask import (Flask, request)
+import config
+import settings
+import json
+from flask_cors import CORS, cross_origin
+from models import PeopleInteract, People
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import func
+import smtplib
+
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = '0c483b7de838a087dac0263a96dc3b7b'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite://///home/alberto.rios/globant-mossions/flask-projects/cora-app/api.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
+
+
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+db = SQLAlchemy(app)
+
+
+@app.route('/login', methods=['POST'])
+@cross_origin()
+def login():
+    if request.method == 'POST':
+        username = settings.users.get(request.form['username'] ,None)
+        if not username:
+            return {'login':'user no valid'}
+        if username.Password == request.form['password']:
+            return {'login':True}
+        
+        return {'login':'password invalid'}
```

Comment:
Why do we have just one file with doing all the work? 
I mean, this file works as an endpoint, validates data, interacts with a db.

Reply:
Hi @crisvlean , you are right, I have been working in  a new version with better organization and documentation using swagger , in next hours I will push a new PR with the changes. 

Thanks

Reply:
Got it. Please let me know to review it.